### PR TITLE
TE-3.7: Base Hierarchical NHG Update: ExplicitIPv6EnableForGRIBI deviation

### PR DESCRIPTION
**Pull request by stthummuri**
Added Ipv6 entries for all dut and ate ports

### CODE REVIEW

File: feature/gribi/otg_tests/base_hierarchical_nhg_update/base_hierarchical_nhg_update_test.go [hunk 1]
```diff
@@ -70,35 +70,47 @@ var (
 		Desc:    "dutPort1",
 		IPv4:    "192.0.2.1",
 		IPv4Len: 30,
+		IPv6:    "2001:db8::192:0:2:1",
+		IPv6Len: 126,
```
Comment (sachendras) [inline feature/gribi/otg_tests/base_hierarchical_nhg_update/base_hierarchical_nhg_update_test.go:74]: Should all these be updated to match the ate equivalent?
https://github.com/openconfig/featureprofiles/blob/4b62cb22e7cff45cbb8f697af3f94a9232e6d6ce/feature/gribi/ate_tests/base_hierarchical_nhg_update/base_hierarchical_nhg_update_test.go#L66-L96

Comment (octpetre) [reply feature/gribi/otg_tests/base_hierarchical_nhg_update/base_hierarchical_nhg_update_test.go:74]: Not sure why IPv6 interfaces are needed here as the traffic is IPv4 only

Comment (mohanasm) [reply feature/gribi/otg_tests/base_hierarchical_nhg_update/base_hierarchical_nhg_update_test.go]: Similar to changes in ate_tests. Refer: https://github.com/openconfig/featureprofiles/pull/859

Comment (stthummuri) [reply feature/gribi/otg_tests/base_hierarchical_nhg_update/base_hierarchical_nhg_update_test.go]: > Not sure why IPv6 interfaces are needed here as the traffic is IPv4 only

Removed ipv6 interfaces

**Review by sachendras — APPROVED** [review]

**Pull request merged**